Lint failed:

```
make[1]: Entering directory '/home/circleci/project/receiver/awsecscontainermetricsreceiver'
golangci-lint run --allow-parallel-runners
golangci-lint run --allow-parallel-runners
WARN [runner] Can't run linter goanalysis_metalinter: exportloopref: failed prerequisites: [errors in package: [/home/circleci/project/receiver/awscontainerinsightreceiver/factory_test.go:24:2: could not import go.opentelemetry.io/collector/testbed/testbed (/home/circleci/go/pkg/mod/go.opentelemetry.io/collector@v0.27.1-0.20210520162255-f7674b2f593e/testbed/testbed/receivers.go:36:2: could not import go.opentelemetry.io/collector/receiver/prometheusreceiver (/home/circleci/go/pkg/mod/go.opentelemetry.io/collector@v0.27.1-0.20210520162255-f7674b2f593e/receiver/prometheusreceiver/factory.go:21:4: could not import github.com/prometheus/prometheus/discovery/install (/home/circleci/go/pkg/mod/github.com/prometheus/prometheus@v1.8.2-0.20210430082741-2a4b8e12bbf2/discovery/install/install.go:19:4: could not import github.com/prometheus/prometheus/discovery/aws (/home/circleci/go/pkg/mod/github.com/prometheus/prometheus@v1.8.2-0.20210430082741-2a4b8e12bbf2/discovery/aws/ec2.go:29:2: could not import github.com/aws/aws-sdk-go/service/ec2 (-: could not load export data: no export data for "github.com/aws/aws-sdk-go/service/ec2")))))]] 
ERRO Running error: exportloopref: failed prerequisites: [errors in package: [/home/circleci/project/receiver/awscontainerinsightreceiver/factory_test.go:24:2: could not import go.opentelemetry.io/collector/testbed/testbed (/home/circleci/go/pkg/mod/go.opentelemetry.io/collector@v0.27.1-0.20210520162255-f7674b2f593e/testbed/testbed/receivers.go:36:2: could not import go.opentelemetry.io/collector/receiver/prometheusreceiver (/home/circleci/go/pkg/mod/go.opentelemetry.io/collector@v0.27.1-0.20210520162255-f7674b2f593e/receiver/prometheusreceiver/factory.go:21:4: could not import github.com/prometheus/prometheus/discovery/install (/home/circleci/go/pkg/mod/github.com/prometheus/prometheus@v1.8.2-0.20210430082741-2a4b8e12bbf2/discovery/install/install.go:19:4: could not import github.com/prometheus/prometheus/discovery/aws (/home/circleci/go/pkg/mod/github.com/prometheus/prometheus@v1.8.2-0.20210430082741-2a4b8e12bbf2/discovery/aws/ec2.go:29:2: could not import github.com/aws/aws-sdk-go/service/ec2 (-: could not load export data: no export data for "github.com/aws/aws-sdk-go/service/ec2")))))]] 
make[1]: *** [../../Makefile.Common:93: lint] Error 3
make[1]: Leaving directory '/home/circleci/project/receiver/awscontainerinsightreceiver'
make: *** [Makefile:144: for-all-target-./receiver/awscontainerinsightreceiver] Error 2
make: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/home/circleci/project/receiver/awsecscontainermetricsreceiver'
```

Most likely need to run `make gotidy`

--- a/cmd/otelcontribcol/components.go
+++ b/cmd/otelcontribcol/components.go
@@ -31,6 +31,7 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/f5cloudexporter"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/honeycombexporter"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/humioexporter"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influxdbexporter"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerthrifthttpexporter"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter"
@@ -160,6 +161,7 @@ func components() (component.Factories, error) {
 		f5cloudexporter.NewFactory(),
 		googlecloudexporter.NewFactory(),
 		honeycombexporter.NewFactory(),
+		humioexporter.NewFactory(),
 		influxdbexporter.NewFactory(),
 		jaegerthrifthttpexporter.NewFactory(),
 		loadbalancingexporter.NewFactory(),

--- a/go.mod
+++ b/go.mod
@@ -16,6 +16,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/f5cloudexporter v0.0.0-00010101000000-000000000000
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.0.0-00010101000000-000000000000
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/honeycombexporter v0.0.0-00010101000000-000000000000
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/humioexporter v0.0.0-00010101000000-000000000000
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influxdbexporter v0.0.0-00010101000000-000000000000
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerthrifthttpexporter v0.0.0-00010101000000-000000000000
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter v0.0.0-00010101000000-000000000000
@@ -108,6 +109,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dynat
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/f5cloudexporter => ./exporter/f5cloudexporter
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/honeycombexporter => ./exporter/honeycombexporter
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/humioexporter => ./exporter/humioexporter
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influxdbexporter => ./exporter/influxdbexporter
 

